docs/releases/updating_ez_platform.md

```diff
@@ -212,29 +212,36 @@ index 49a17a9..80c4cd7 100644
 
     If you are upgrading to v1.8 you need to create this folder and assign it to a new specific Section. Then, add them in the config (for example, in `app/config/default_parameters.yml`, depending on how your configuration is set up):
 
-    ``` bash
+    ​``` bash
         #Location id of the root for form uploads
         form_builder.upload_folder.location_id: <folder location id>
-
+    
         #Section identifier for form uploads
         form_builder.upload_folder.section_identifier: <section identifier>
-    ```
+    ​```
 
 **Increased password hash length**
 
 !!! caution "V1.12"
 
     v1.12.0 improves password security by introducing support for PHP's `PASSWORD_BCRYPT` and `PASSWORD_DEFAULT` hashing algorithms. By default `PASSWORD_DEFAULT` is used. This currently uses bcrypt, but this may change in the future as PHP adds support for new and stronger algorithms.
+    Apply the following database update script to change the schema and enable the storage of longer passwords:
+    Note that the script is available for PostGreSQL as well.
```

Comment:
`PostgreSQL` with small "g"
Also, right now this sounds like the same script also works for Postgres, so maybe:
`Note that there is a script available for PostgreSQL as well.`